customer_outstanding_statement/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================================
+Print Customer Outstanding Statement
+====================================
+
+The outstanding statement provides details of all outstanding customer receivables
+up to a particular date. This includes all unpaid invoices, unclaimed refunds and
+outstanding payments. The list is displayed in chronological order and is split by currencies.
+
+Aging details can be shown in the report, expressed in aging buckets (30 days
+due, ...), so the customer can review how much is open, due or overdue.
+
+Configuration
+=============
+
+Users willing to access to this report should have proper Accounting & Finance rights:
+
+#. Go to *Settings / Users* and edit your user to add the corresponding access rights as follows.
+#. In *Application / Accounting & Finance*, select *Billing* or *Billing Manager*
+#. In *Technical Setting* mark *Show Full Accounting Features* options.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Invoicing > Master Data > Customers and select one or more
+#. Press 'Action > Customer Outstanding Statement'
+#. Indicate if you want to display aging buckets
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/91/11.0
+
```

Comment:
@hveficent Can you add a Roadmap, that will be good to have a action that will print the report, for all partners that have due amounts, at a date specified, will be more easy, if you have a large amount of customers, you will have to select, let's say 5000 in the tree view to print for all which are not due, that's why adding a menu in reporting, and in partner_ids to get all partners, not the active_ids..

Reply:
This Roadmap is not needed. It's a cool idea to implement, and maybe I will do it just after the two PRs get merged. Thanks for the idea.